thinc/backends/ops.py

```diff
@@ -978,6 +978,35 @@ def backprop_hard_swish_mobilenet(
             return dY
         return dX * dY
 
+    def dish(self, X: FloatsType, inplace: bool = False) -> FloatsType:
```

Comment:
I assume you probably want to update with `FloatsXdT` now that https://github.com/explosion/thinc/pull/720 is merged?

Reply:
Yeah, I'll merge in master and update this PR.

Reply:
Done.